extensions/mapstore/src/main/java/com/hazelcast/mapstore/GenericMapStore.java

```diff
@@ -297,7 +297,9 @@ public void destroy() {
 
         asyncExecutor.submit(() -> {
             awaitInitFinished();
-            dropMapping(mapping);
+            if (instance.node.isRunning()) {
```

Comment:
Why don't you add a new isRunning() method to HazelcastInstanceImpl class :-)

Reply:
184063b305d